fast-list.js

```diff
@@ -258,6 +299,11 @@ FastList.prototype = {
       );
     }
```

Comment:
@etiennesegonzac can you tell me more about what the above logic is for? This may be broken now our initial render is `criticalOnly`?